core/test/RectGrid_test.cpp

```diff
@@ -24,13 +24,14 @@
 #include <fstream>
 
 const std::string filename = "RectGrid_test.nc";
-const std::string partition_filename = "partition_metadata_1.nc";
+const std::string partition_filename = "partition_metadata_3.nc";
```

Comment:
could you please send this partition file (`partition_metadata_3.nc`) to @timspainNERSC  so that he can upload it to the ftp server?

That way I can add it to the CI workflow.

My PR #507 `wget`s the required `.nc` files and will run the test as part of CI.

Reply:
Instead I've implemented generation of `partition_metadata_3.nc` file from its text dump during the building process.